### PR TITLE
ceph-volume: fix regression and improve output in lvm list

### PR DESCRIPTION
**Pull request by jan--f**
fix various lvm list issues
    
A single report on a non-lvm device now works.
Format was cleaned up, report lvm journal,wal, db only once.
Fixes: https://tracker.ceph.com/issues/44009

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by jan--f** [comment]
@leseb it doesn't, LVs are expected to be passed by name. If we should change that, a tracker ticket would be a start for discussions.

**Comment by jan--f** [comment]
> @leseb it doesn't, LVs are expected to be passed by name. If we should change that, a tracker ticket would be a start for discussions.
> _edit_: The reasoning is that its not straight forward as we rely on lvm's reporting a lot. This metadata has no notion of which convenience links that lvm creates for LVs.

I take it all back...this works now :)

**Comment by leseb** [comment]
> > @leseb it doesn't, LVs are expected to be passed by name. If we should change that, a tracker ticket would be a start for discussions.
> > _edit_: The reasoning is that its not straight forward as we rely on lvm's reporting a lot. This metadata has no notion of which convenience links that lvm creates for LVs.
> 
> I take it all back...this works now :)

Ok great coz it's working on 14.2.5 :-)

**Comment by jan--f** [comment]
> Ok great coz it's working on 14.2.5 :-)

Yeah sorry, that was a  result of improving some internal code structures (not scanning all lvs on the system multiple times). Though I'd still argue it a was a bug before ;)

In any case please feel free to open bugs and feature requests of you feel anything should be different. I'm trying to keep track of the various issues, but I forget ;)
I prefer closing tickets over forgetting.

**Comment by jan--f** [comment]
jenkins test ceph-volume tox

**Comment by jan--f** [comment]
jenkins test ceph-volume tox

File: src/ceph-volume/ceph_volume/api/lvm.py [hunk 1]
```diff
@@ -1348,6 +1347,15 @@ def create_lvs(volume_group, parts=None, size=None, name_prefix='ceph-lv'):
     return lvs
 
 
+def get_device_lvs(device, name_prefix=''):
```
[the remark this answers is not in the document]
Comment (jan--f) [reply src/ceph-volume/ceph_volume/api/lvm.py]: no, filtering by underlying drive is awkward as lvm tracks drives `/dev/sdb(0)` and an lv report can't be filtered by `pv_name`.

**Pull request merged**